lib/galaxy/webapps/galaxy/services/visualizations.py

```diff
@@ -299,29 +291,27 @@ def _add_visualization_revision(
     def _create_visualization(
         self,
         trans: ProvidesUserContext,
-        type: str,
-        title: Optional[str] = None,
-        dbkey: Optional[str] = None,
-        slug: Optional[str] = None,
-        annotation: Optional[str] = None,
-        save: Optional[bool] = True,
+        type: SanitizedString,
```

Comment:
Is this really `SanitizedString` at runtime ? I would assume it's just a regular string ?